pyeudiw/satosa/default/request_handler.py

```diff
@@ -57,8 +57,10 @@ def request_endpoint(self, context: Context, *args) -> Response:
         }
 
         # load all the trust handlers request jwt header parameters, if any
-        self.trust_evaluator.get_selfissued_jwt_header_trust_parameters(issuer=self.client_id)
 
+        _protected_jwt_headers.update(
+            self.trust_evaluator.get_selfissued_jwt_header_trust_parameters(issuer=self.client_id)
```

Comment:
different trust evaluation handler may have different client ids (unfortunately)

we have the default_client_id that sets each trust ahdnler client id automatically if not differently provided

this means that we should be able to customize a client id within each trust handler configurations

Reply:
An initial support for this feature can be found here:
https://github.com/italia/eudi-wallet-it-python/pull/366/commits/24ae4890fcfea76e1693f024248f5244a49f5dfe